### DISCUSSION TITLE
How can I show rendered HTML instead of the HTML source?

### DISCUSSION THREAD

**Discussion by markfilipak** [community/community#29863] [Other Feature Feedback, Questions, & Ideas] [answered]
I would contact punjoe if I knew how...

I saw punjoe's reply here: 'https://github.com/orgs/community/discussions/29439'.
punjoe cites this page:
'https://punjoe.github.io/puns-developers-site/'
How did punjoe create 'punjoe.github.io'?
How does a github visitor/contributor find that link from this page:
'https://github.com/punjoe/puns-developers-site'?

In my case, my main page is:
'https://github.com/markfilipak/Video-Object-Notation/blob/main/index.html'
but when I browse to it, I get the HTML source code, not a rendered page. So it appears that github is redirecting the https.

Can anyone crack this nut?

Thanks,
Mark Filipak.

**Comment by airtower-luna** [comment, marked as answer]
The post you're citing is about GitHub Pages. Read the [Pages documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) to see how it's done. You have finished HTML files, so what you want is called a *static* site.

> How does a github visitor/contributor find that link from this page:

You give it to them, for example list it as the website for your repository or put a link into the README.

**Reply by markfilipak** [reply]
> The post you're citing is about GitHub Pages. Read the [Pages documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) to see how it's done. You have finished HTML files, so what you want is called a _static_ site.

Three-quarters of the way though that page is
`In the "Code and automation" section of the sidebar, click Pages.`
Done.
`To see your published site, under "GitHub Pages", click your site's URL.`
Not found. The documentation appears to be stale.

I tried browsing to 'markfilipak.github.io' and got a 404.
I also tried browsing to 'markfilipak.github.io/video-object-notation' and got a 404.

**Reply by radar-blip** [reply]
It seems you have it working now?

https://markfilipak.github.io/Video-Object-Notation/

**Reply by markfilipak** [reply]
Yes! (Apparently I needed to refresh the page.)
Thanks Folks!

**Comment by delosemre** [comment]
You need to create a[ Github page](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages). 
Simply create a markfilipak.github.io repository and upload your HTML+CSS files. 
Or just provide the view with readme.md.

**Comment by the-m-files** [comment]
Wait, how do you transfer files from one branch to another. and then create a static website.
